### DISCUSSION TITLE
Proposal: <podcast:country> tag

### DISCUSSION THREAD

**Discussion by nathangathright** [Podcastindex-org/podcast-namespace#616] [Tag Proposal]
As suggested by @benjaminbellamy and @samsethi, here’s a quick writeup of what I’ve heard others suggest.

### Purpose
The [`<podcast:location>`](https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#location) tag describes the location of editorial focus for a podcast's content, and the [`<language>`](https://www.rssboard.org/rss-specification#optionalChannelElements:~:text=language) tag describes the language the channel is written in, but a new `<podcast:country>` tag would describe the country of origin for the podcast.

### Parent
`<channel>`

### Count
Single

### Node Value
This is a free-form string meant to be a human-readable location. This value cannot be blank. Please do not exceed `128 characters` for the node value, or it may be truncated by aggregators.

### Attributes
* **code:** (recommended) A two-letter country code as defined by the [ISO 3166-1 alpha-2](https://www.iso.org/iso-3166-country-codes.html)

### Examples
```xml
<podcast:country code="GB">United Kingdom</podcast:country>
```

**Comment by nathangathright** [comment]
My only hesitation is that [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) excludes England, Scotland, and Wales. For that, we’d have to support [ISO 3166-2 subdivisions](https://en.wikipedia.org/wiki/ISO_3166-2), which seems overly specific for all other use cases.

**Reply by jamescridland** [reply]
Putting aside the (in my opinion) uselessness of this tag overall - why are we only "recommending" this code, not mandating it?

I say this as someone who was simultaneously born in

* England
* Great Britain
* The United Kingdom

...and you have no idea how tiresome it is to look in country drop-downs under "E", "G", "T" and then "U" every time someone asks you for your birth country.

**Comment by johnspurlock** [comment]
Curious: what would the `podcast:country` value be for the podnews weekly review show?

**Comment by jamescridland** [comment]
Please could I understand the use-case? Why would this be a useful additional field to burden podcasters with? How does it help the listener find a podcast that she would like to listen to? How might a sample UX look?

as @johnspurlock hints at - the country is at best immaterial for Podnews’s daily show, and at worst, downright misleading. It is not a show about Australia. Why is it of any consequence where the show is from?

A show from Canada could be in at least three different languages; Switzerland might be in four or five. So a listener that only speaks Romansch is not going to have much fun with a “view podcasts from Switzerland” navigation?

The language tag is widely supported and helps a listener find shows in a language she speaks.

The location tag helps a listener find a show that is “about” the Eiffel Tower or the Brooklyn Bridge. That show could have been made in Malaysia or Vanuatu; that’s immaterial to the content.

What is the problem that this tag seeks to solve? And why is it best solving it with a country code and not a mirror of the location tag’s capabilities of specifying a country, a city, a street or a building?

**Reply by jamescridland** [reply]
@nathangathright I'd really appreciate a response to "why is it best solving it with a country code and not a mirror of the location tag’s capabilities of specifying a country, a city, a street or a building?" - it seems to me that, if we have to produce something like this, we do so using a rather more flexible way than just a "country".

**Reply by nathangathright** [reply]
> As suggested by @benjaminbellamy and @samsethi, here’s a quick writeup of what I’ve heard others suggest.

I have no interest in advocating for this tag beyond creating this discussion thread. Multiple parties seemed interested but hadn’t bothered writing anything up, so I did just to put it to bed, one way or another. Que Será, Será.

**Comment by samsethi** [comment]
We have implemented the podcast:country tag.  We have it in every podcasters admin dashboard.  We then show the country flag in the search results, the podcast page and on our homepage i.e we show the top 10 podcasts by country.  As a user I can change my country to see other podcasts #discovery

We are going to implement a maps solution.  So people can discover where a podcast exists.  I would like the location tag to be used conjunction with the country tag to further specific a location.  So it might be (Country) Australia, (Location) Brisbane.  Then if I wanted to do a meetup in a location I can see a map of all the podcasters. e.g Maps.fm (now closed). 

However we use the country location in many places on TrueFans. 

As a podcaster I can set my own country in the admin. 

<img width="1148" alt="Cursor_and_TrueFans___Dashboard" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/1bd910ac-977e-41a1-acc5-ce60c8318b0d">

TrueFans Homepage - we show the Top 10 podcasts for the country the user selected in their settings. 

<img width="1418" alt="Cursor_and_TrueFans___For_The_Fans™" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/053a1751-5935-41f5-b191-ab9eb1520ab9">

User Settings - you can change the country to see the top 10 of different countries.

<img width="1277" alt="TrueFans___Settings" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/53ddc487-2fa7-4cca-8fc1-96079d5d4417">

Search - You can also search by country.  We want to include a boolean search for country and location. 
<img width="1285" alt="Cursor_and_TrueFans___Search_for_Podcasts_or_Fans" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/5aa52705-2694-4611-8276-54f1989e1f9e">

**Reply by keunes** [reply]
This makes me think of the country-specific suggestion lost that Apple provides, which is cool, except that it presents popular shows based on the consumer side (what is popular in country X) rather than the producer side (what is popular whose producers are from country X). For Discovery, as a consumer, I find the former more interesting than the latter.

> I would like the location tag to be used conjunction with the country tag to further specific a location

This makes it sound like you want to move away from the location tag's initial purpose: from what the podcast is about to a more detailed specification of where the podcast is produced. It would be a shame to lose the location functionality even further (it allegedly already is 'misused' in many cases).

**Reply by jamescridland** [reply]
> We are going to implement a maps solution. So people can discover where a podcast exists. I would like the location tag to be used conjunction with the country tag to further specific a location. So it might be (Country) Australia, (Location) Brisbane.

This is just dumb, Sam, sorry.

The use-cases for the location tag (which is about where the podcast is **about** ):
* I love railways and I'd like to find every podcast about railway stations
* I am going to Leeds next week. What podcasts are about places 15km from the centre of Leeds?
* I am a tourist and looking forward to flying to Sydney next week. What podcasts are there about tourist attractions in Sydney?

Because the location tag is connected to OpenStreetMap, it allows a user interface to do just that - as well as the slightly less exciting "podcasts on a map" that maps.fm produced.

I still can't understand the use-case for the "country" tag (i.e. "where is this podcast recorded").
* I can record a podcast about Washington DC from my office in Brisbane. What problem are we solving for the consumer to highlight that this podcast about Washington DC was recorded in Brisbane?
* The example you give, the Podnews Weekly Review, is normally recorded in the UK and in Australia. It discusses global news and regularly doesn't cover things happening in the UK or in Australia. So why slap an Aussie flag on it? It shouldn't have one on there.

If your use-case is "show me shows which are popular in GB" then you don't do that by looking at where shows are *made*, you do that by looking at where shows are *consumed*. And you alone have that aggregate data.

If your use-case is "show me shows which are about things in GB" then use the location tag.

If your use-case is "show me shows in my language" then use the language tag.

Or if your use-case is none of those things, please explain what your use-case is.

**Reply by picard102** [reply]
> What problem are we solving for the consumer to highlight that this podcast about Washington DC was recorded in Brisbane?

The problem of the user wanting to find podcasts recorded by people in Brisbane. 

> Or if your use-case is none of those things, please explain what your use-case is.

Show me other podcasts made by people in Australia. 
Show me popular Canadian podcasts.

**Reply by jamescridland** [reply]
> The problem of the user wanting to find podcasts recorded by people in Brisbane.

Where a podcast is recorded is of no interest to a listener if the content covered is not relevant to them. I've yet to see a compelling use-case for a listener, which is all I really care about. A podcast's recording location conveys almost nothing about the content of the show. I'm pushing against this because I don't believe listeners are asking to find "podcasts recorded near me". (I *do* believe they're looking for the similar "podcasts recorded about places near me".)

**Reply by picard102** [reply]
> Where a podcast is recorded is of no interest to a listener 

As a listener for over 18 years I have often looked for podcasts local to me to listen too. I know others who have as well.

**Reply by podmirror** [reply]
Hi James, @picard102 I am Fyddo John researching podcasts in India. I was trying to use **podcastindex.org** to write an academic paper on the rise of regional podcasts. In India, we have 18 major languages and as many Indians are settled abroad the country tag can help differentiate between those in foreign lands and those podcasts/podcasting from India. I agree with James on a listener may not be interested in geotags. But it can help content creators and researchers like us who are exploring this cultural phenomenon taking off. I was also seeking an advice of whether can we use this podcastindex data for research work. Cause I see practically no one using this data for research. kindly advice

**Comment by samsethi** [comment]
If no one else wants a country tag then so be it.  Its part of TrueFans and we have some more ideas on how we plan to use the country metadata.  As shown above we already have several ways in which we use the country flag.

**Reply by picard102** [reply]
I think there are people who want it and would find a use case for it. I also think it would be valuable for platforms that will be required to surface country specific content in the near future.

**Reply by PofMagicfingers** [reply]
I can see some usecases myself, combining other tags or ideas you could search for podcasts from a specific country to get a cultural/exotic/local point of view about a subject or anything :
- what podcast from France are telling about this Turkish movie?
- which podcast from the USA has done an episode about my city?
-what nationalities are the podcasts I'm listening to?

Of course sometimes a show hasn't a country. Sometimes it doesn't make sense. But some times it does. I guess it can be an optional tag the creator would decide if they need it or not.

**Reply by benjaminbellamy** [reply]
@samsethi, you are not alone. 🫶

**Comment by theDanielJLewis** [comment]
I think this tag is totally unnecessary because it's redundant.

Before you eat me alive, please hear me out!

We already have `<podcast:location>`. All it would take to support this "origin" idea _and_ clarify how the location tag can be used would be an attribute.

This is what that location tag looks like currently:

```xml
<podcast:location geo="geo:30.2672,97.7431" osm="R113314">Austin, TX</podcast:location>
```

I suggest that, because of precedence, we assume that a tag as written above is the location the podcast or episode are _about_.

But we add an extra attribute:

```xml
<podcast:location geo="geo:30.2672,97.7431" osm="R113314" subject="true">Austin, TX</podcast:location>
```

Now, this tag explicitly indicates that this location is a subject of the episode.

Or switch it:

```xml
<podcast:location geo="geo:30.2672,97.7431" osm="R113314" origin="true">Austin, TX</podcast:location>
```

Now, the exact same location tag and generator can be used to indicate that this location is not the subject of the content, but its origin.

I was originally thinking we should support `rel="subject"` and `rel="origin"`, but then that would get redundant when the subject and origin are the same location.

This approach would make implementation and parsing easier than adding an additional tag, and it has the added bonus of further clarifying the existing tag.

**Reply by theDanielJLewis** [reply]
And if all you want to indicate is the country, then a simple `country` attribute could do that just fine, and would work for both subject and origin uses of the `location` tag.

**Reply by Marzal** [reply]
+1 to improve existing tags instead of creating new ones. And +1 to this particular use case.

I like the rel="usecase" version, it seems simpler to parse, but I would ask [the current parties](https://podcastindex.org/apps?elements=Location) supporting the tag location

**Reply by Dwev** [reply]
I like this idea.

Not all tags are for the direct  benefit of the listener; some are for the benefit of producers, hosts, advertisers, and others.

Adding an attribute to the existing tag addresses what "country" is trying to solve, and allows for variable fidelity. It also means that if there are cases where the origin location doesn't make sense (e.g. podcasters in different countries for example) then you don't have to include it.

**Reply by theDanielJLewis** [reply]
I suppose we could do `rel="subject,origin"` if the same location was _both_ the subject and the origin.

> It also means that if there are cases where the origin location doesn't make sense (e.g. podcasters in different countries for example) then you don't have to include it.

I think that's where we should allow multiple instances of this tag. And I only _just now_ realized that `location` is a single-use tag, but I don't think it should be. Episodes could be about multiple locations. For example, maybe someone is doing an episode about the best Zoos on the east coast, and so they need to tag multiple locations.

**Reply by Dwev** [reply]
You can include location within the chapter JSON though, so that supports the multiple locations you mentioned, as long as you also use chapters.

**Reply by jamescridland** [reply]
I like this. How do I say "recorded in Orlando, but this is about the Brooklyn Bridge"? Two tags?

**Comment by benjaminbellamy** [comment]
Hello everyone!
I'm just starting to read this thread now, wow. I didn't expect so much enthusiasm and passion. Thank you @nathangathright for starting this.
I read carefully everything you all wrote. I do not agree with everything that was written. 😉

**TLDR;**
I still think we need a `podcast:country` tag with iso-3166 that would tell **from which country the podcast is from**.

- `podcast:country`  has nothing to do with the `podcast:location` tag. Please do not mix apples 🍏 with kimchi 🍜. The `podcast:location` tag is here to specify which location **you are talking about**. It is about content. `podcast:country` is **not about content**, it is **about container**. Having two different things here would make it difficult to parse, especially since `podcast:location` is already used for one purpose.
- `podcast:country` can be useful for **users looking for podcasts from specific countries**. For instance, as a french speaker, I'd really love to find podcasts specifically from Québec ⚜️, or Côte d'Ivoire 🇨🇮, or Belgium 🇧🇪… or France 🇫🇷.
- `podcast:country` could be really helpful to know **how many podcasts there are per country**. 📊
- `podcast:country` could be really helpful in terms of **analytics** (hello OP3 @johnspurlock !). Knowing the total amount of downloads for french speaking podcasts is not really useful. Knowing the total amount of downloads for podcasts from France would be. 📈
- **Advertisers** who are dealing with podcasters would be able to **target a country**. 💸
- **Regulations** are different for every country. Knowing the country of origin of a podcast can help knowing which ones apply. 👩‍⚖️
- A simple country tag with [ISO 3166-1 alpha-2](https://www.iso.org/iso-3166-country-codes.html) is the most simple and efficient way. 🚀

**Reply by picard102** [reply]
I think modifying the location tag is more useful here as it allows a more detailed location than just the country, and it makes use of a tag that's not really well used anyways. 

Adding an attribute to it clears up any parsing issues with a flag for origin and a default/flag for subject. 

While the ISO code is a bit easier to parse, it's not very granular.

**Reply by benjaminbellamy** [reply]
I do not agree. 😉
I don't see real use for granularity here (it's nowhere in my list).
We need country of origin, that's enough.
And please, I'm begging you, DO NOT MIX APPLE WITH WITH KIMCHI!

**Reply by benjaminbellamy** [reply]
If the location tag is not used (spoiler: that's not true, but let's assume it is), then throw it away and make a new one. (And call it `podcast:country` for instance). Recycle your garbage, not your podcast tags. ♻️

**Reply by picard102** [reply]
I mean I'm far more interested as a listener in seeing podcasts from my town/city than I am the country as a whole. Allowing more detail doesn't really take away from the overarching goal of identifying the country.

**Reply by benjaminbellamy** [reply]
I get your point but I do not think it's worth it.
Keep it simple.
I would not makes things over complicated to everyone for 5% of users. Because the risk when a feature gets really complex is that in the end no one implements it. And no one uses it.

**Reply by picard102** [reply]
If the location tag already exists, then modifying it would be the least complex solution, and easiest to implement. If we don't want to complicate things.

**Reply by benjaminbellamy** [reply]
No, no, no and **NO**.
- No it would not least complex solution.
- I developed the `podcast:location` in Castopod. It would be **much** easier to add a new `podcast:country` than tweaking an already existing feature that was **never** designed for that purpose.
- It would be **much** easier to have a two character country code.
- It would be easier not to mix apples and kimchi.

**Reply by PofMagicfingers** [reply]
I second that. Modifying the location tag could lead to confusion. If you add a `rel` attribute, it would be ignored by old implementations, some `rel=origin` would be misinterpreted as a classic location tag. 


Also kimchi and apple, `podcast:country` seems suited for quick triage with a standard code and that info is a good duo with the language. Like @benjaminbellamy said, many countries speak French, knowing which country the podcast itself want to be associated with is a good thing.

In fact, one of the reason we didn't published an annual top listened podcasts at podCloud, is the lack of triage between podcasts from french Quebec, French African countries, and French european countries. It's not the same audience at all, and it would be nice to sort those top listened by countries of "podcast nationality".

And as stated before, all tags are optional, so a podcast could decide to let it empty if it is an international show.

**Reply by samsethi** [reply]
I want to support both country and location.  Country is great for many reasons to identify the number of podcasts per country etc.  We use it also to identify the top 10 podcasts in each country.  We use the GEOlocation API to set a default location for users and then in their settings they can change the country if they wish.  I will be supporting the two letter iso code and this will stay in TrueFans as a feature even if its not supported by the community.  We have other ideas on what we want to do with the country field.  

The location tag is more specific.  Country could be UK but my location is Marlow, Bucks (shown in GEO code). Again I think the location filed has failed to date when its used to talk about the location discussed in the podcast.  However there are two use cases location where I am located and location of the topic discussed in the podcast.  @theDanielJLewis suggestion is a good one to help use one field to have multiple functions.  

I want to be able to map locations of podcasts to show local communities of podcasters.

**Comment by samsethi** [comment]
@nathangathright we added ISO 3166 support to our search engine.  So now when you choose the "Country" filter you can just type the two letter country code or type out the full country name.  

 
<img width="1271" alt="TrueFans___Search_for_Podcasts_or_Fans_🔊" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/80f6478f-3cc0-4e82-ad75-8779a86eb45a">
